src/Stan/Config.hs

```diff
@@ -146,6 +147,9 @@ configToCliCommand ConfigP{..} = "stan " <> T.intercalate " \\\n     " (map chec
         CheckScopeDirectory dir -> " --directory=" <> toText dir
         CheckScopeModule m -> " --module=" <> unModuleName m
 
+defaultChecks :: [FilePath] -> HashMap FilePath (HashSet (Id Inspection))
+defaultChecks = HashMap.fromList . map (, inspectionsIds)
+
```

Comment:
Since it's a function, not a single value, would it make sense to rename it to `mkDefaultChecks`?
Also, I guess it can be used in the `applyChecks` function now :+1: